server/env.ts

```diff
@@ -40,8 +39,8 @@ const env = cleanEnv(process.env, {
   MAIL_PASSWORD: str(),
   REPORT_EMAIL: str({ default: "" }),
   CONTACT_EMAIL: str({ default: "" }),
-  SENTRY_PRIVATE_DSN: str({ default: "" }),
-  SENTRY_PUBLIC_DSN: str({ default: "" })
+  BUGSNAG_API_KEY: str({ default: "57e3d9dc3851fa10ed31e323cfef834d" }),
```

Comment:
I would consider not having the API key baked in here so that if someone forks our fork we don't get unrelated noise to our Bugsnag project.